kokoro/scripts/test/go_test.sh

```diff
@@ -213,7 +208,11 @@ if [[ "${SHORT:-false}" == "true" ]]; then
 fi
 
 TEST_UNDECLARED_OUTPUTS_DIR="${LOGS_DIR}" \
-  go test -v "${TEST_SUITE_NAME}.go" \
+  gotestsum \
+  "${GOTESTSUM_ARGS:-}" \
```

Comment:
This is failing with: `flag provided but not defined: -packages`

I think your GOTESTSUM_ARGS being `''` is messing it up.

I recommend skipping gotestsum_args for this CL and we can discuss what you want GOTESTSUM_ARGS to do in a follow up CL.

Reply:
Done